### ISSUE TITLE
Feature request: Nightscout-connect support

### ISSUE THREAD

**Issue by DunkyDaMonkey** [marciogranzotto/addon-nightscout#20]
I couldn't figure out how to turn on nightscout-connect (https://github.com/nightscout/nightscout-connect) as an internal plugin. Its entirely possible I'm being daft. 

When I get a bit more time, I might try as a sidecar deployment. 

The aim would be to be able to pull down our data from Glooko (which is experimental on the nightscout-connect) for CGM backfill and also the insulin basal/bolas and food that is all pushed to Glooko.

**Comment by i-cz** [comment]
Hi @DunkyDaMonkey have you been able to figure it out ? I am trying the similar thing using Librelinkup .. I presume we have to somehow set the environment variable

**Comment by i-cz** [comment]
@marciogranzotto do you have any hint for us please?

**Comment by DunkyDaMonkey** [comment]
Nope. I looked at all glooko apis and rewrote them using requests in python, I could not get any of them to authenticate and provide me data without a developer code that glooko won't give out. 

I went a different route, where I put xdrip and camaps fx on a follower, and got it to push glucose values to my nightscout server.  It does mean I cannot get insulin values to populate, so thay prevents me from doing anything fancier than calculating time in range figures, deviation and deltas.

Xdrip has been pretty flawless.
